### ISSUE TITLE
Add support of Event API

### ISSUE THREAD

**Issue by bradh11** [CiscoDevNet/ciscosparkapi#55]
I would like to see support of Cisco Spark event API endpoint to be added.

**Comment by cmlccie** [comment]
Yes, I had noticed here the other night that the API endpoints had been added to the docs.  I'll get these added as soon as possible.

**Comment by cmlccie** [comment]
@bradh11 it's been a busy past several weeks, but I did finally get around to adding support for the Cisco Spark Events API _(included in v0.10+)_.  The API is fully tested and documented, see the [Events API](http://ciscosparkapi.readthedocs.io/en/latest/user/api.html#events) and [Event data model](http://ciscosparkapi.readthedocs.io/en/latest/user/api.html#event) docs for details.

Take a look and let me know if you have any issues.
